spec/models/original_record_spec.rb

```diff
@@ -147,9 +147,10 @@
 
     it 'reloads content_type' do
```

Comment:
Exactly, that's what we're testing.  We want the `content_type` to sync from Marmotta's entry on reload.  The behavior of the test doesn't change here, it's just not hard coded anymore.